R/app.R

```diff
@@ -120,7 +120,7 @@
 #' to \code{true}, or a timeout happens. It returns \code{TRUE} is the
 #' expression evaluated to \code{true}, possible after some waiting.
 #'
-#' \code{find_widget()} finds the corresponding HTML element of a Shiny
+#' \code{app$find_widget()} finds the corresponding HTML element of a Shiny
 #' widget. It returns a \code{\link{widget}} object.
 #'
 #' \code{expect_update()} is one of the main functions to test Shiny apps.
```

Comment:
I wasn't sure if to add "app$" to expect_update or not.  From what I can tell, `app$expect_update(...) === expect_update(app, ...)`, so the latter is just a "simplified" version? It feels a bit weird that all the other app methods are methods of the app class, while this one is its own function.  Is it because you expect it to be the most commonly used function?